<launch>

	<node name="gf_node_29" pkg="geofrenzy" type="gf_node" args="29" respawn="true" />
	<node name="gf_node_59" pkg="geofrenzy" type="gf_node" args="59" respawn="true" />
	<node name="gf_node_74" pkg="geofrenzy" type="gf_node" args="74" respawn="true" />
	<node name="gf_node_70" pkg="geofrenzy" type="gf_node" args="70" respawn="true" />
	<node name="gf_node_71" pkg="geofrenzy" type="gf_node" args="71" respawn="true" />
	<node name="gf_node_72" pkg="geofrenzy" type="gf_node" args="72" respawn="true" />

	<node name="GFServer_70" pkg="geofrenzy" type="gf_map_server" args="70" respawn="true" />
	<node name="GFServer_71" pkg="geofrenzy" type="gf_map_server" args="71" respawn="true" />
	<node name="GFServer_72" pkg="geofrenzy" type="gf_map_server" args="72" respawn="true" />
	<node name="GFServer_73" pkg="geofrenzy" type="gf_map_server" args="74" respawn="true" />


	<node name="GPS" pkg="nmea_navsat_driver" type="nmea_serial_driver" args="_port:=/dev/nd-105-c _baud:=4800" respawn="true"/>

  <node pkg="swri_transform_util" type="initialize_origin.py" name="initialize_origin">
    <param name="local_xy_frame" value="map"/>
    <param name="~local_xy_origin" value="auto"/>
  </node>

	<node pkg="tf" type="static_transform_publisher" name="base_link_base_laser_broadcaster" args="0.1 0 0.1 0 0 0  base_link base_laser 100" />
	<node pkg="tf" type="static_transform_publisher" name="map_to_odom" args="0 0 0 0 0 0 map odom  100" />
	<node pkg="tf" type="static_transform_publisher" name="map_to_local_origin" args="0 0 0 0 0 0 map local_origin  100" />
	<node pkg="tf" type="static_transform_publisher" name="fcu_to_base_link" args="0 0 0 0 0 0 fcu base_link 100" />



</launch>
